src/Test/PHPUnit/AbstractCacheConstraint.php

```diff
@@ -62,6 +62,6 @@ protected function matches($other)
      */
     protected function failureDescription($other)
     {
-        return (string) $other . ' ' . $this->toString();
+        return 'the responses are identical';
```

Comment:
PHPUnit will render this as ‘failed asserting that…’.

Unlike Guzzle 3, PSR-7’s `ResponseMessage` has no `__toString()`, so we have to do it ourselves. The problem is that just dumping out the full response message (headers and body) can make the failure description quite illegible. Should we show just the headers? Showing only the cache/miss header is not very useful either: if you assert it’s a miss and it’s a hit it’s quite obvious that the header says miss. :wink: